### ISSUE TITLE
Replace phusion's baseimage

### ISSUE THREAD

**Issue by ciprianc** [ministryofjustice/postcodeinfo#67]
Our container is using phusion's baseimage image and is running 2 of our processes (nginx and gunicorn) and a few from phusion (like cron,syslog). We initially went this way to easier serve static assets, but we should either serve them with gunicorn or from S3. This is problematic because the logs generated are mixed (json and non json) as multiple processes can log. Also, arguably we don't need that many things running inside one container. We want lightweight single process containers, not virtual machines implemented in cgroups.

A alternative to phusion is https://registry.hub.docker.com/_/python/ or even alpine.

**Comment by lukaszraczylo** [comment]
Still relevant - @aldavidson please advise.

**Comment by aldavidson** [comment]
Yeah, it's still there: https://github.com/ministryofjustice/postcodeinfo/blob/master/Dockerfile#L1

Not urgent, but we should sort this if we get time to
